common/persistence/dataStoreInterfaces.go

```diff
@@ -942,6 +943,14 @@ func (d *DataBlob) GetEncodingString() string {
 	return string(d.Encoding)
 }
 
+// GetData is a safe way to get the byte array or nil
+func (d *DataBlob) GetData() []byte {
```

Comment:
is this method needed? Why is `nil` not safe?

Reply:
it's used as a helper for getting data safely